### PR TITLE
Include time unit in metadata proto fields.

### PR DESCRIPTION
**Pull request by mansu**
Include time unit in metadata proto fields so it's easier to add time in a consistent format everywhere.

### CODE REVIEW

File: kaldb/src/main/java/com/slack/kaldb/chunk/ChunkInfo.java [hunk 1]
```diff
@@ -23,11 +23,11 @@
   public static ChunkInfo fromSnapshotMetadata(SnapshotMetadata snapshotMetadata) {
     return new ChunkInfo(
         snapshotMetadata.snapshotId,
-        snapshotMetadata.startTimeUtc,
-        snapshotMetadata.endTimeUtc,
-        snapshotMetadata.startTimeUtc,
-        snapshotMetadata.endTimeUtc,
-        snapshotMetadata.endTimeUtc,
+        snapshotMetadata.startTimeEpochMsUtc,
```
Comment (bryanlb) [inline kaldb/src/main/java/com/slack/kaldb/chunk/ChunkInfo.java:26]: If we're going with "epoch" as part of the name we don't need to include "UTC" as well. 

I do like including the units in the name so it's abundantly clear.

**Review by bryanlb — CHANGES_REQUESTED** [review]
I'd like to see us remove the "UTC" portion of the names as this is redundant with "epoch." 
We also should include the other proto definitions as well in `kladb_search` as they have yet another time unit format.

File: kaldb/src/main/java/com/slack/kaldb/chunk/ChunkInfo.java (hunk 1 above)
Comment (mansu) [reply kaldb/src/main/java/com/slack/kaldb/chunk/ChunkInfo.java]: Good call. I removed the utc from the names.

**Comment by mansu** [comment]
This code is ready for re-review. Let's handle the other protos in a separate diff.

**Review by bryanlb — APPROVED** [review]

**Pull request merged**